spikingjelly/activation_based/examples/dsqn/train.py

```
@@ -56,7 +56,7 @@ def eval_Q(env, net, cuda):
                 rand_actions = np.random.choice(n_actions, sum(mask))
                 actions[mask] = rand_actions
 
-                obs, reward, done, _ = env.step(actions)
+                obs, reward, done, _ = env.step(actions.item())
```

Ruff (0.16.1)

[warning] Unpacked variable `reward` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)